docs/instruments/ZJSONInstrument_v0.0.1.md

```diff
@@ -0,0 +1,1028 @@
+It is important to set the default language as it is used as a key in other locations of the `schema` object. This key is required to be compatible with
+ReproNim's JSON-LD Instrument.
+
+`meta.derivedFrom`: String.
+                    
+The resource instrument that the Z-JSON Instrument derived from as a result of significant content modification such that the instrument is no longer the same                    
+instrument, and the creation of a new version does not suffice.
+                    
+This level of modification does not include format transcription i.e. linst to
+JSON, unless the content of the instrument has also been modified. The
+`meta.importedFrom` key should be used in the case of conversions.
+
+`meta.importedFrom`: String.
+
+The resource instrument that this Z-JSON Instrument was converted from, given
+that the imported content is preserved, and only its format is modified.
+
+If the Z-JSON instrument is an original instrument, the value of this key is
+null.
+
+`meta.includeMetaDataFields`: Boolean. 
+
+True of false that the Z-JSON Instrument will include LORIS-specific
+metadata fields, such as 'Candidate Age', or 'Examiner', which
+calculates the candidate's age, and selects the examiner
+administering the instrument, respectively.  
+
+`meta.instrumentVersion`: Required string.
+
+The version of the instrument being described.
+
+It follows some convention as determined by the author of the instrument,
+starting with some representation of the number 1 and incrementing itself alongside any changes made to the `schema` object. It is self-contained and
+interpreted independently of the instrument's `shortName` or `longName`.
+
+`meta.lastUpdateOn`: String.
+
+The date at which small modifications, such as bug fixes, that do not justify
+a new version be created, was last made.
+                          
+`meta.longName`: Required object with IETF BCP 47 language tags as keys.
+
+The value of each key will be the long, human readable version of the instrument's name
+- its title - provided in every language that the instrument comes in.
+
+`meta.multilingual`: Boolean.
+
+True or false that the instrument is available in more than one language that is
+its `defaultLanguage`.
+
+`meta.previousVersion`: String.
+
+The previous version of the Z-JSON Instrument, if significant changes
+were made to justify creating this new version of the instrument.
+
+This may be redundant metadata but it may also be helpful in showing the
+lineage of existing instruments. If changes to an instrument is so
+significant that they no longer share the same lineage, `meta.derivedFrom`
+would be used to track changes. 
+
+`meta.schemaVersion`: Required string.
+
+The version of this Z-JSON Instrument Specification document that the
+instrument complies with. It follows the format of v*SpecVersion*-dev i.e. `v0.0.1-dev` if following this
```

Comment:
it could be defined as semver, thats not a bad idea. i'll need some help defining the versions because i'm not totally confident in my knowledge of this